clients/src/components/UserFavouritesCard.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+// image
+import eleven from '../assets/eleven.jpg';
+
+const UserFavouritesCard = (props) => (
+  <div>
+    <div className="col-md-4">
+      <div className="card">
+        <div className="card-header">{props.name}</div>
+        <div className="card-main">
+          <img src={eleven} alt="Recipe Image" className="img-thumbnail" />
+          <div className="main-description">
+            <p> {props.description} </p>
```

Comment:
React components should always have one parent component. If not, an error is thrown. I can see you solved this problem using a `div`, which is fine 👍 . In React 16, a component called `Fragment` was introduced to serve this purpose. Please replace the `div`s with this component. This would greatly cleanup your code.

You can read more about them here .
[React Fragments](https://reactjs.org/docs/fragments.html)

Reply:
Thank you for the feedback.